src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -41,9 +46,23 @@ class ConfirmationPage extends React.Component {
     const response = form.submission.response
       ? form.submission.response.attributes
       : {};
-    const { veteranFullName, verified } = form.data;
+    const {
+      first: firstName = '',
+      middle: middleName = '',
+      last: lastName = '',
```

Comment:
Setting some defaults here so I don't have to check if exists when calling `toUpperCase()` later.